helm-charts/splunk-otel-collector/templates/config/_otel-collector.tpl

```diff
@@ -245,9 +265,32 @@ service:
     {{- end }}
 
     {{- if or (eq (include "splunk-otel-collector.logsEnabled" .) "true") (eq (include "splunk-otel-collector.profilingEnabled" .) "true") }}
+    {{- if (eq (include "splunk-otel-collector.splunkO11yEnabled" .) "true") }}
+    logs/split:
+      receivers: [otlp]
+      exporters: [routing/logs]
+    {{- end }}
+    {{- if (eq (include "splunk-otel-collector.splunkO11yEnabled" .) "true") }}
```

Comment:
Double trouble? Can lines 272 and 273 be removed?

Reply:
Fixed